src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/PageType.php

```diff
@@ -74,6 +74,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ),
                 'required' => false,
             ])
+            ->add('use_combination_image_in_listing', SwitchType::class, [
+                'label' => $this->trans(
+                    'Use combination image in listings',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'This option allows you to choose which image you want to use in listings for products with combinations. By default, it always uses the cover image of the product. If you enable this option and your filtering module is properly passing required information, you can show directly show the image of this combination.',
```

Comment:
```suggestion
                    'This option allows you to choose which image to display in listings for products with combinations. By default, the cover image of the product will be used. If you enable this option and your filtering module is properly passing the required information, then the default image of the combination found will be displayed.',
```

Reply:
Hello 👋 
I think the wording you added to the PR description is better so I copy pasted that one. Just a few things to edit for clarity. Thanks!